NanoTasks/nanotasks/src/main/java/com/nanotasks/BackgroundWork.java

```diff
@@ -1,5 +1,26 @@
+/*
+* Copyright (C) 2015 Fabien Devos.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
 package com.nanotasks;
 
+/**
+ * A interface that's have a generic type and
+ * throws a Exception
```

Comment:
The grammar is not correct but more importantly, that comment is not useful at all.